tools/ldb_cmd.cc

```diff
@@ -2634,10 +2662,13 @@ void BackupCommand::DoCommand() {
   }
   printf("open db OK\n");
   std::unique_ptr<Env> custom_env_guard;
-  Env* custom_env = NewCustomObject<Env>(test_cluster_, &custom_env_guard);
+  Env* custom_env = NewCustomObject<Env>(backup_env_uri_, &custom_env_guard);
   BackupableDBOptions backup_options =
-      BackupableDBOptions(test_path_, custom_env);
-  backup_options.max_background_operations = thread_num_;
+      BackupableDBOptions(backup_dir_, custom_env);
+  if (stderr_log_level_ != InfoLogLevel::NUM_INFO_LOG_LEVELS) {
+    backup_options.info_log = new StderrLogger(stderr_log_level_);
```

Comment:
yes, thanks!

BTW, does anyone know what's wrong with the sandcastle logs? I see "output truncated" in the logs and no obvious failure messages.